tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Connecting a Haskell Backend to a PureScript Frontend
+published: not-yet
+ghc: 7.10.3
+purs: 0.10.5
+lts: 7.17
+libraries: purescript-bridge
+language: haskell purescript
+author-name: Javier Casas Velasco
+description: In this tutorial we will implement a way to extend the types in the Haskell backend to the PureScript frontend while maintaining consistency and simplifying communication.
+---
+# Connecting a Haskell Backend to a PureScript Frontend
+## Introduction
+At [Stack Builders](https://www.stackbuilders.com) we are working on a full-stack app with [CollegeVine](https://www.collegevine.com/) using Functional Languages.
```

Comment:
Mohan approved the tutorial:

> Looks great! I wouldn't say that I was worried you were going to say something negative, 
but more that I wanted to see the tone and language of the article since we don't really talk about our engineering publicly. Overall we are happy to be part of this great discovery! :)